twisted/plugins/docker_xylem_plugin.py

```diff
@@ -24,6 +25,9 @@ class DockerServiceMaker(object):
     def makeService(self, options):
         config = yaml.load(open(options['config']))
 
+        if not os.path.exists('/run/docker/plugins'):
+            os.makedirs('/run/docker/plugins')
+
```

Comment:
We probably don't want to do this unless our socket is configured to live there.